src/Microsoft.DocAsCode.Build.Engine/TemplatePreprocessors/TemplateJintPreprocessor.cs

```diff
@@ -53,14 +53,16 @@ public class TemplateJintPreprocessor : ITemplatePreprocessor
         private const string RequireFuncVariableName = "require";
         private const string RequireRelativePathPrefix = "./";
 
+        private const string NullString = "null";
+
         private object _utilityObject;
         private static readonly object ConsoleObject = new
         {
-            log = new Action<object>(s => Logger.Log(s)),
-            info = new Action<object>(s => Logger.LogInfo(s.ToString())),
-            warn = new Action<object>(s => Logger.LogWarning(s.ToString())),
-            err = new Action<object>(s => Logger.LogError(s.ToString())),
-            error = new Action<object>(s => Logger.LogError(s.ToString())),
+            log = new Action<object>(s => Logger.Log(s ?? NullString)),
```

Comment:
the change is to make the behavior of console.log the same as browser / node.js, with console.log(null) logs "null"